src/evolutionaryAlgorithm/SimulationComponents/FitnessFunction/FitnessFunctionFactory.py

```diff
@@ -0,0 +1,22 @@
+from src.evolutionaryAlgorithm.SimulationComponents.FitnessFunction import FitnessFunctionInterface
+from src.evolutionaryAlgorithm.SimulationComponents.FitnessFunction.impl.weierstrass import weierstrass
+from src.evolutionaryAlgorithm.SimulationComponents.FitnessFunction.impl.expandedSchaffers import expandedSchaffers
+from src.evolutionaryAlgorithm.SimulationComponents.SimulationComponentFactoryInterface import \
+    SimulationComponentFactoryInterface
+
+
+class FitnessFunctionFactory(SimulationComponentFactoryInterface):
+    @staticmethod
+    def validate(config: dict):
+        pass
+
+    @staticmethod
+    def build(config: dict) -> FitnessFunctionInterface:
+
+        fitnessFunction = config["FitnessFunction"]
+        if fitnessFunction["Type"] == "weierstrass":
+            return weierstrass()
+        elif fitnessFunction["Type"] == "expandedSchaffers":
+            return expandedSchaffers()
+        else:
+            raise NotImplementedError()
```

Comment:
Hej to nie jest do niczego używane chyba, w jaki sposub tworzysz individuum używając info o fitnes function

Reply:
nie do końca rozumiem, w momencie gdy robiliśmy sobie evaluate to było pomocne

Reply:
masz na myśli get evaluation w individualu? No niezbyt bo to nie jest nijak czytane bo individual ma publiczny konstruktor